src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -128,6 +129,38 @@ trait BackendCall {
     throw new NotImplementedError(s"resume() called on a non-resumable BackendCall: $this")
   }
 
+  def useCachedCall(cachedBackendCall: BackendCall)(implicit ec: ExecutionContext): Future[ExecutionHandle] = ???
+
+  /**
+   * Compute a hash that uniquely identifies this call
+   */
+  def hash: String = {
+    val orderedInputs = locallyQualifiedInputs.toSeq.sortBy(_._1)
```

Comment:
There might be a bug here...

```
task t {
  File f
  command { ps }
}
```

versus

```
task t {
  File f
  String s = "static"
  command { ps }
}
```

Assuming the same value for `f`, they'll job avoid to each other.  `locallyQualifiedInputs` in both cases will only have one entry, for `f`.  Since the command is not affected by `String s`, it gets left undetected.

I'm not sure if this is a bug or not.